docs/nv2-sign-verify-aks.md

```diff
@@ -2,15 +2,20 @@
 
 ## Install the notation cli and azure-kv plugin
 
-> NOTE: These are temporary steps for early development. Officially released binaries will be available as the latest changes are merged.
+> NOTE: The walkthrough uses pre-released versions of notation, notation plugins and ratify.  
 
-1. Build a version of notation extensibility support
+1. Install notation with plugin support from <https://github.com/notaryproject/notation/releases/tag/feat-kv-extensibility>
 
     ```bash
-    git clone git@github.com:notaryproject/notation.git -b feat-kv-extensibility
-    cd notation
-    make build
-    
+    # Choose a binary
+    timestamp=20220121081115
+    commit=17c7607
+
+    # Download, extract and install
+    curl -Lo notation.tar.gz https://github.com/notaryproject/notation/releases/download/feat-kv-extensibility/notation-feat-kv-extensibility-$timestamp-$commit.tar.gz
+    tar xvzf notation.tar.gz
+    tar xvzf notation_0.0.0-SNAPSHOT-${commit}_linux_amd64.tar.gz -C ~/bin notation
+        
     # Copy the notation cli to your bin directory
     cp ./bin/notation ~/bin
```

Comment:
This should be `cp ~/bin/notation /usr/local/bin` or be removed since you export to ~/bin